tests/entrypoints/openai/test_oot_registration.py

```diff
@@ -18,11 +17,8 @@
 
 class MyOPTForCausalLM(OPTForCausalLM):
 
-    def compute_logits(
-        self,
-        hidden_states: torch.Tensor,
-        sampling_metadata: SamplingMetadata,
-    ) -> Optional[torch.Tensor]:
+    def compute_logits(self, hidden_states: torch.Tensor,
+                       sampling_metadata: SamplingMetadata) -> torch.Tensor:
```

Comment:
any particular reason for this change? it's Optional in the superclass method that it's overriding...

Reply:
this is automatically done by the linter, I think due to the version change of linter.